app/controllers/data_controller.rb

```diff
@@ -37,18 +37,28 @@ def index
 ) AS geojson from (" + @sites.to_sql+ ") AS subquery1) AS subquery2").to_sql)[0]['measurements'], adapter: nil, serializer: nil        
       }
       format.csv {
-        query = "COPY (SELECT * FROM data_views WHERE id IN (" + @data.xrons.pluck(:id).join(', ').to_s + ") ) TO STDOUT WITH CSV HEADER"
-        
+        query = <<~SQL
+          COPY (
+            SELECT DISTINCT "id", "labnr", "bp", "std", "cal_bp", "cal_std", "delta_c13", "source_database", "lab_name", "material", "species", "feature", "feature_type", "site", "country", "lat", "lng", "site_type",
+                   periods::text AS periods,
+                   typochronological_units::text AS typochronological_units,
```

Comment:
What does this refer to? There's no such table (or shouldn't be).

Reply:
This comes in the materialized view from typos, stays there and in the csv for legacy reasons. Problem?